homeassistant/components/zwave_js/api.py

```diff
@@ -976,13 +982,9 @@ async def websocket_validate_dsk_and_enter_pin(
     {
         vol.Required(TYPE): "zwave_js/provision_smart_start_node",
         vol.Required(ENTRY_ID): str,
-        vol.Exclusive(
-            PLANNED_PROVISIONING_ENTRY, "options"
-        ): PLANNED_PROVISIONING_ENTRY_SCHEMA,
-        vol.Exclusive(
-            QR_PROVISIONING_INFORMATION, "options"
-        ): QR_PROVISIONING_INFORMATION_SCHEMA,
-        vol.Exclusive(QR_CODE_STRING, "options"): QR_CODE_STRING_SCHEMA,
+        vol.Required(QR_PROVISIONING_INFORMATION): QR_PROVISIONING_INFORMATION_SCHEMA,
```

Comment:
I suggest we remove dead code in a separate PR.

Do you know why the dead code is here, and what changed in the frontend to not use it?

Reply:
I don't think it was ever used. Or the add device flow changed to not use it but it must have been long ago. And that flow is chaning again now

Reply:
If I don't remove the extra options here, I'd have to support getting the dsk from the other option formats which would needlessly complicate this PR

Reply:
Can we make a separate PR that we merge first, where we remove the dead code?